Test/CoreSDK.Test/Shared/Extensibility/MetricManagerTest.cs

```diff
@@ -150,8 +150,8 @@ public void AggregatedMetricTelemetryIntervalDurationPropertyIsPositiveInteger()
             Assert.Equal(1, aggregatedMetric.Count);
             Assert.Equal(1, aggregatedMetric.Properties.Count);
 
-            Assert.True(aggregatedMetric.Properties.ContainsKey("Microsoft.ApplicationInsights.Metrics.Aggregation.IntervalMs"));
-            Assert.True(long.Parse(aggregatedMetric.Properties["Microsoft.ApplicationInsights.Metrics.Aggregation.IntervalMs"]) > 0);
+            Assert.True(aggregatedMetric.Properties.ContainsKey("Microsoft.AppInsights.Metrics.Aggregation.IntervalMs"));
```

Comment:
you should refrain from using `AppInsights`. `ApplicationInsights` is the correct term